src/main/java/io/nats/client/impl/NatsJetStreamImplBase.java

```diff
@@ -47,7 +47,7 @@ protected ConsumerInfo _getConsumerInfo(String streamName, String consumer) thro
         return new ConsumerInfo(resp).throwOnHasError();
     }
 
-    protected ConsumerInfo _createConsumer(String streamName, ConsumerConfiguration config) throws IOException, JetStreamApiException {
+    ConsumerInfo _createConsumer(String streamName, ConsumerConfiguration config) throws IOException, JetStreamApiException {
```

Comment:
private?

Reply:
I need access within the package to this.